### DISCUSSION TITLE
A configurable "last accessed/modified" property

### DISCUSSION THREAD

**Discussion by PastelStoic** [geldata/gel#4431] [Ideas]
I see a lot of users wanting to store when an object was created, last modified, or last accessed. I think it would be a great QOL feature to have this built in to the schema. Based on user configuration, the "last modified/accessed" properties could be set to update on access/change of the object itself, or of any number of linked objects. "Created at" is probably the easiest - it can be done now with a simple readonly field + default value, only requiring a shorter syntax. The modify/access would probably require pub/sub functionality, which I'm aware is in progress.

**Comment by Sikarii** [comment]
This is most likely gonna be a breeze with the upcoming triggers, which I believe are planned for 3.0.

**Reply by PastelStoic** [reply]
Ah, planned for 3.0? Wish that roadmap update would release soon lol.
